### PR TITLE
Changed the footer for revue newsletter

### PR DESCRIPTION
**Pull request by charlene1804**
Pierrick asked me to change the subscribe button on Backlight's landing to the new newsletter system/ url REVUE

See associated PR in webcomponents-editor https://github.com/divriots/webcomponents-editor/pull/3225

### CODE REVIEW

**Comment by gqio** [comment]
Copy paste from DRC ?

**Comment by charlene1804** [comment]
Pretty much, keeping the original style in tailwind.

File: footer/src/index.vue [hunk 1]
```diff
@@ -88,70 +98,70 @@
...
+import Logo from '../../logo/src/logo.vue';
+
+export default {
+  props: {
+    routes: { type: Array, default: [] },
+    subscribe: { type: String, required: true },
+    contact: { type: String, required: true },
+  },
+  data() {
+    const params = new URL(this.subscribe).searchParams;
```
Comment (gqio) [inline footer/src/index.vue:114]: Still used ? I don't think so. You could remove the whole function/block of `data()`

Comment (charlene1804) [reply footer/src/index.vue]: I wasn't sure. Done and it seems to change nothing/ work fine.

Comment (charlene1804) [reply footer/src/index.vue]: Done.

**Review by gqio — APPROVED** [review]

**Pull request merged**